build-logic/binary-compatibility/src/main/groovy/gradlebuild/EnrichedReportRenderer.groovy

```diff
@@ -70,7 +70,15 @@ class EnrichedReportRenderer extends GroovyReportRenderer {
                         result.acceptedApiChanges.push(correction);
                     });
                     // Sort the array in place by type, then member
-                    result.acceptedApiChanges = result.acceptedApiChanges.sort((a, b) => (a.type +'#' + a.member) > (b.type + '#' + b.member));
+                    // Note that Firefox is fine with a sort function returning any positive or negative number, but Chrome 
+                    // requires 1 or -1 specifically and ignores higher or lower values
+                    result.acceptedApiChanges = result.acceptedApiChanges.sort((a, b) => { 
```

Comment:
🧐 : no need to set `result.acceptedApiChanges`, `Array.sort()` sorts in-place.

Reply:
Good catch.  I even commented that it did this, and at some point stopped doing it...